src/geo/map/torque-layer.js

```diff
@@ -31,6 +31,10 @@ var TorqueLayer = MapLayer.extend({
   },
 
   getInfowindowData: function() {
+    var infowindow = this.get('infowindow');
```

Comment:
there is a lot of repeated code between the cartodb layers and torque, we may want to have a "interactivelayer" base class to have this kind of things